staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -78,13 +78,13 @@ func ValidateLabels(labels map[string]string, fldPath *field.Path) field.ErrorLi
 func ValidateDeleteOptions(options *metav1.DeleteOptions) field.ErrorList {
 	allErrs := field.ErrorList{}
 	if options.OrphanDependents != nil && options.PropagationPolicy != nil {
-		allErrs = append(allErrs, field.Invalid(field.NewPath(""), options, "OrphanDependents and DeletionPropagation cannot be both set"))
+		allErrs = append(allErrs, field.Invalid(field.NewPath("propagationPolicy"), options.PropagationPolicy, "OrphanDependents and DeletionPropagation cannot be both set"))
```

Comment:
lower case "orphanDependents" and "propagationPolicy"

Reply:
Done.